test/integration/authorization_test.go

```diff
@@ -595,7 +625,7 @@ func TestAuthorizationResourceAccessReview(t *testing.T) {
 				Users:           sets.NewString("edgar"),
 				Groups:          sets.NewString(),
 				Namespace:       "mallet-project",
-				EvaluationError: `role "admin" not found`,
```

Comment:
it will require `Group: ""` here: https://github.com/mfojtik/origin/blob/2259dd5022d7ff7af22d53d4d3214e935c19bdf5/test/integration/authorization_test.go#L552